godotproject/Player.gd

```diff
@@ -79,7 +80,7 @@ func start_swing():
 
 func stop_swing():
 	# Convert swinging angular momentum into player velocity
-	velocity = Vector2.DOWN.rotated(swing_angle) * swing_angular_speed * swing_radius
+	#velocity = Vector2.DOWN.rotated(swing_angle) * swing_angular_speed * swing_radius
```

Comment:
I comment this out because I didn't think it made sense to have this happen if we're using the launch button. I can't actually tell what it does though. It doesn't change the motion if its in or commented out

Reply:
Hmm... this manually recalculates the player's velocity when they exit a swing. It is working the same whether it's in or not since they would already have the velocity that we used last frame, so this should basically not affect the velocity at all. I would actually say that we should take this out and just put a comment saying we're using the value from last frame.
```suggestion
	# ... this is actually unnecessary, since we already have the velocity from last frame.
	# We'll use that instead to avoid any bugs with additional velocity sources.
```